config.analysis

```diff
@@ -1,19 +1,20 @@
 [case]
 # ACME case configuration
-casename = 20160805v0atm.A_WCYCL1850_v0atm.ne30_oEC.edison.alpha7_00
+casename = 20160805v0atm.A_WCYCL1850_v0atm.ne30_oEC.edison.alpha8
 native_res = ne30
 short_term_archive = 0
 ref_casename_v0 = B1850C5_ne30_v0.4
 
 [paths]
 # paths to simulation and observational datasets
-archive_dir = /scratch1/scratchdirs/petercal/ACME_simulations
-archive_dir_ocn = /scratch1/scratchdirs/petercal/ACME_simulations/20160805v0atm.A_WCYCL1850_v0atm.ne30_oEC.edison.alpha7_00/run
-scratch_dir = /global/project/projectdirs/acme/xylar/20160805v0atm.A_WCYCL1850_v0atm.ne30_oEC.edison.alpha7_00.test.pp
-plots_dir = /global/project/projectdirs/acme/xylar/coupled_diagnostics_20160805v0atm.A_WCYCL1850_v0atm.ne30_oEC.edison.alpha7_00-20160520.A_WCYCL1850.ne30_oEC.edison.alpha6_01
-log_dir = /global/project/projectdirs/acme/xylar/coupled_diagnostics_20160805v0atm.A_WCYCL1850_v0atm.ne30_oEC.edison.alpha7_00-20160520.A_WCYCL1850.ne30_oEC.edison.alpha6_01.logs
+archive_dir = /global/cscratch1/sd/golaz/ACME_simulations
+archive_dir_ocn = /global/cscratch1/sd/golaz/ACME_simulations/20160805.A_WCYCL1850.ne30_oEC.edison.alpha7_00/run/ 
+scratch_dir = /global/project/projectdirs/acme/lvroekel/20160805v0atm.A_WCYCL1850_v0atm.ne30_oEC.edison.alpha7_00.test.pp
+plots_dir = /global/project/projectdirs/acme/lvroekel/coupled_diagnostics_20160805v0atm.A_WCYCL1850_v0atm.ne30_oEC.edison.alpha7_00-20160520.A_WCYCL1850.ne30_oEC.edison.alpha6_01
+log_dir = /global/project/projectdirs/acme/lvroekel/coupled_diagnostics_20160805v0atm.A_WCYCL1850_v0atm.ne30_oEC.edison.alpha7_00-20160520.A_WCYCL1850.ne30_oEC.edison.alpha6_01.logs
```

Comment:
I don't think we want to change the defaults here, although it probably doesn't fully matter as long as particular entries aren't removed.  However, it would be easy in the future to accidentally remove entires.  @xylar should comment on this too.